app/provider_details/rest.py

```diff
@@ -19,8 +20,17 @@
 
 @provider_details.route('', methods=['GET'])
 def get_providers():
-    data = provider_details_schema.dump(get_provider_details(), many=True).data
-    return jsonify(provider_details=data)
+    provider_details = provider_details_schema.dump(get_provider_details(), many=True).data
```

Comment:
It could be more efficient to do this in one query. So add provider_details to the the stats query.

Reply:
@servingUpAces I've changed this so that it now uses one new DAO query to get all the data